cli/flox/src/commands/mod.rs

```diff
@@ -918,6 +922,7 @@ impl InternalCommands {
             InternalCommands::Auth(args) => args.handle(config, flox).await?,
             InternalCommands::Build(args) => args.handle(config, flox).await?,
             InternalCommands::Publish(args) => args.handle(config, flox).await?,
+            InternalCommands::LockManifest(args) => args.handle(flox).await?,
```

Comment:
(non-blocking): `flox lock` feels more natural to me than `flox lock-manifest`.

Reply:
Yeah, I did wonder too, but we can always change it later if it's hidden/private.